packages/react-components/src/components/chart/hooks/useXAxis.ts

```diff
@@ -0,0 +1,18 @@
+import { ChartAxisOptions, ViewportInMs } from '../types';
+import { DEFAULT_X_AXIS } from '../eChartsConstants';
+import { XAXisOption } from 'echarts/types/dist/shared';
+import { useMemo } from 'react';
+
+export const useXAxis = (viewportInMs: ViewportInMs, axis?: ChartAxisOptions): XAXisOption => {
+  return useMemo(
+    () => ({
+      show: axis?.showX ?? DEFAULT_X_AXIS.show,
+      type: 'time' as const,
+      splitNumber: 6,
+      interval: 1000 * 60, // 1sec interval
```

Comment:
does this number work for all relative durations? Why does it have to be 1 minute?

Reply:
good catch - i was playing around this value, I dont think we should be setting this and let echarts scale it however it does , will remove